dev/tasks/tasks.yml

```
@@ -1588,9 +1588,9 @@ tasks:
       image: conda-python-hdfs
 {% endfor %}
 
-{% for python_version, spark_version, test_pyarrow_only, numpy_version in [("3.8", "v3.1.2", "false", "latest"),
-                                                                           ("3.9", "v3.2.0", "false", "1.23"),
-                                                                           ("3.10", "master", "false", "latest")] %}
+{% for python_version, spark_version, test_pyarrow_only, numpy_version in [("3.8", "v3.4.1", "false", "latest"),
+                                                                           ("3.10", "v3.4.0", "false", "1.23"),
```

Should we use the same version (`v3.4.1` or `v3.4.0`) here?

---

I thought on covering several versions but now that I look into it again, a patch difference will not make a huge difference so I am ok changing it to:
```suggestion
{% for python_version, spark_version, test_pyarrow_only, numpy_version in [("3.8", "v3.4.1", "false", "latest"),
                                                                           ("3.10", "v3.4.1", "false", "1.23"),
```